pytorch_lightning/core/lightning.py

```diff
@@ -35,6 +34,7 @@
 from pytorch_lightning.core.mixins import DeviceDtypeModuleMixin, HyperparametersMixin
 from pytorch_lightning.core.optimizer import LightningOptimizer
 from pytorch_lightning.core.saving import ModelIO
+from pytorch_lightning.plugins import DDPSpawnPlugin
```

Comment:
within the framework let's try to use the full import path. maybe that will address the issue you're seeing?